lib/watir-webdriver/browser.rb

```diff
@@ -10,6 +10,7 @@ class Browser
     include Waitable
 
     attr_reader :driver
+    attr_reader :checkers
```

Comment:
Since we're already changing the API itself then maybe we could use some better naming here as well instead of ambigious `checker`? What about something in the lines of `after_hooks` or something similar, because that's what they essentially are. Or?